src/parser/jobs/brd/modules/AdditionalStats.js

```diff
@@ -387,6 +429,11 @@ export default class AdditionalStats extends Module {
 				}
 			}
 		}
+		// If there are no damage instances, we can't really calculate K. Let's trigger brokenLog and return 1
+		if (!values || !values.length) {
+			this.brokenLog.trigger()
```

Comment:
Wary of triggering `brokenLog` here - I mean, no damage in a log is _weird_, but not necessarily _broken_. I think returning a sane baseline K is reasonable enough.